run_coverage.sh

```
@@ -0,0 +1,48 @@
+# ================================
+# 1. 运行 fuzzer
+# ================================
+./build/linux/x86/release/RyanJson \
+  ./test/fuzzer/corpus \
+  -dict=./test/fuzzer/RyanJsonFuzzer.dict \
+  -timeout=4 \
+  -runs=99999999 \
+  -verbosity=0 \
+  -max_len=8192 \
+  -workers=5 \
+  -jobs=10 \ 
```

Shellcheck (0.11.0)

[error] Delete trailing spaces after \ to break line (or use quotes for literal space).

(SC1101)